<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listar Pessoas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(80, 161, 241);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#ffffff;&quot;&gt;LISTA DE PESSOAS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>641</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CPF-CNPJ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sobrenome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Endereço</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Bairro</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cidade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Telefone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Usuário</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Senha</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btExcluirPessoas">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>560</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>EXCLUIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSairPessoas">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>560</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOrdenarPorPessoas">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>ORDENAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btGerarRelatorioPessoas">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
</string>
    </property>
    <property name="text">
     <string>RELATÓRIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Apresentação1-removebg-previewaawwwwwwwwwwww.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btExcluirPessoas_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>EDITAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
